docs/Changelog.md

```diff
@@ -10531,6 +10531,49 @@ This version of the operator has been available since version 11 of the default
 <dd>Constrain output types to any tensor type.</dd>
 </dl>
 
+### <a name="ConcatFromSequence-11"></a>**ConcatFromSequence-11**</a>
+
+  Concatenate a sequence of tensors into a single tensor.
+  All input tensors must have the same rank.
```

Comment:
Can I stack [3, 2]-tensor and [5, 4]-tensor? Do you have any constraints in sequence elements' shapes?

Reply:
No, I believe the constraints are the same with np.concatenate. This is taken from the Concat spec. I'll update it then.